server/src/main/java/org/spine3/server/entity/EntityRepository.java

```diff
@@ -198,7 +199,7 @@ public E apply(@Nullable Map.Entry<I, EntityStorageRecord> input) {
      * @return all the entities in this repository passed the filters.
      */
     @CheckReturnValue
-    public ImmutableCollection<E> findAll(EntityFilters filters) {
+    public ImmutableCollection<E> findAll(EntityFilters filters, @Nullable FieldMask fieldMask) {
```

Comment:
In fact, we have now a default overload of this method `findAll()` (with no parameters). Do we really want to have 3 or even 4 of them, like just for each case?